File: PHP/bucles while.php
<?php
while (true) {
    echo "Introduzca un valor mayor a 10:\n";
    $a = intval(trim(fgets(STDIN)));  // Leer entrada del usuario

    if ($a > 10) {
        echo "Es correcto\n";
        break;  // Salir del bucle si el valor es correcto
    } else {
        echo "Es incorrecto, pruebe de nuevo\n";
        // No usar break aquí, para permitir que el bucle continúe
    }
}
?>

File: python/Bucles while.py
Contador=0
while Contador<30:
    Contador += 1
    print('iteracion', Contador)
#Combinado con if else
while True:
    print('introduzca  un valor mayor a 10')
    a = int(input())
    if a > 10:
        print('Es correcto')
    else:
        print('Es incorrecto, pruebe de nuevo')
        break
        